lib/components/narrative/line-itin/line-itinerary.js

```diff
@@ -19,23 +19,25 @@ export const LineItineraryContainer = styled.div`
 export default class LineItinerary extends NarrativeItinerary {
   static contextType = ComponentContext
 
-  _headerText () {
+  _headerText() {
```

Comment:
Doesn't appear to be used, so we be can delete this and also `_getSummary` and their deps.

Reply:
Removed in 6d5bc90e